introduction.md

```diff
@@ -3,3 +3,4 @@ Dojo Rules
 
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.